index.html

```diff
@@ -255,13 +258,14 @@ <h1>Skills</h1>
 
 <footer class="footer">
 <section class="contact-me">
-  <h1 class="contact-me-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+  <h3 class="contact-me-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
 
   <form action="https://formspree.io/f/mjvlqgpq" method="POST" class="form">
+        <label for="name">Full name</label>
         <input class="form-list" type="text" id="name" name="username" placeholder="Full Name" required maxlength="30">
-     
+        <label for="email">Email</label>
         <input class="form-list" type="email" id="email" name="email" placeholder="Email Address" required>
-   
+        <label for="textarea">Comment</label>
         <textarea class="form-textarea" name="message" id="textarea" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure you use the **Enter text here** text in a `placeholder="...."` attribute for best practices at is required for the user to every time delete the text itself before he wrote his message. Also, remember to style it according to the Figma design, especially for the `font-family` and `color`